bot/cogs/fun.py

```diff
@@ -261,28 +261,29 @@ async def why(self, ctx: Context) -> None:
 
     @command(aliases=["rhash", "robothash", "rh", "rohash"])
     async def robohash(self, ctx: Context, *, meme: str) -> None:
-        """text => robot image thing."""
+        """Text => robot image thing."""
         try:
-            e = Embed(colour=0x690E8)
+            embed = Embed(colour=0x690E8)
             meme = urllib.parse.quote_plus(meme)
-            e.set_image(url=f"https://robohash.org/{meme}.png")
-            await ctx.send(embed=e)
-        except Exception as e:
-            await ctx.send(f"Something Broke. LOL [{e!s}]")
+            embed.set_image(url=f"https://robohash.org/{meme}.png")
+            await ctx.send(embed=embed)
+        except Exception as error:
+            # What is {error!s} ?
+            await ctx.send(f"Something Broke. LOL [{error!s}]")
 
     async def get_answer(self, ans: str) -> str:
+        """Format an answer."""
         if ans == "yes":
             return "Yes."
-        elif ans == "no":
+        if ans == "no":
             return "NOPE"
-        elif ans == "maybe":
+        if ans == "maybe":
             return "maaaaaaybe?"
-        else:
-            return "Internal Error: Invalid answer LMAOO"
+        return "Internal Error: Invalid answer LMAOO"
 
     @command(aliases=["shouldi", "ask"])
     async def yesno(self, ctx: Context, *, question: str) -> None:
-        """Why not make your decisions with a bot?"""
+        """Why not make your decisions with a bot?."""
```

Comment:
I don't like the idea of having both `?` and `.` both are indicators for sentence ends, there is no reason to combine them

Reply:
First line of docstring should end with a period, not `?`
We gotta rewrite this docstring

Reply:
I agree, we should end with `.` but it doesn't make sense to force this even if the current docstring ends with `?` since `?.` doesn't make any sense so yes, rewriting would be the best option.

Reply:
Only issue is that I have no idea what to put there

Reply:
```suggestion
        """Let the bot answer a yes/no question for you."""
```
This should be sufficient

Reply:
Good idea